From e7f6b5c1a0405b3956a806eb9ca0c97c13392131 Mon Sep 17 00:00:00 2001
From: Jason Edson <jaysonedson@gmail.com>
Date: Wed, 9 Dec 2020 21:37:20 -0800
Subject: [PATCH] combo: Add cortex-a76 to known v8-2a cores

Allow building a76 with TARGET_2ND_ARCH_VARIANT set as armv8-2a

Change-Id: I620aa16d6cc563687d35916fc0cd56fb08f9ca71
Signed-off-by: Pranav Vashi <neobuddy89@gmail.com>
Signed-off-by: Saikrishna1504 <saikrishna26918@gmail.com>
---
 core/combo/TARGET_linux-arm.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/combo/TARGET_linux-arm.mk b/core/combo/TARGET_linux-arm.mk
index 11c19444be..44cc99a946 100644
--- a/core/combo/TARGET_linux-arm.mk
+++ b/core/combo/TARGET_linux-arm.mk
@@ -33,7 +33,7 @@
 KNOWN_ARMv8_CORES := cortex-a53 cortex-a53.a57 cortex-a55 cortex-a73 cortex-a75 cortex-a76
 KNOWN_ARMv8_CORES += kryo kryo385 exynos-m1 exynos-m2
 
-KNOWN_ARMv82a_CORES := cortex-a55 cortex-a75 kryo385
+KNOWN_ARMv82a_CORES := cortex-a55 cortex-a75 cortex-a76 kryo385
 
 ifeq (,$(strip $(TARGET_$(combo_2nd_arch_prefix)CPU_VARIANT)))
   TARGET_$(combo_2nd_arch_prefix)CPU_VARIANT := generic
